.github/actions/okfn-contributors/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:20
```

Como boa prática venho buscando especificar a fonte (costume de ficar alternando entre docker e podman e ter problemas com a fonte padrão).
```suggestion
FROM docker.io/node:20
```